padd.sh

```diff
@@ -319,7 +305,7 @@ GetNetworkInformation() {
 
   full_hostname=${pi_hostname}
   # does the Pi-hole have a domain set?
-  if ! [ -z ${PIHOLE_DOMAIN+x} ]; then
+  if [ -n "${PIHOLE_DOMAIN+x}" ]; then
```

Comment:
```suggestion
  if [[ -n "${PIHOLE_DOMAIN+x}" ]]; then
```

Use double brackets for consistency. I know that's not what you changed but since you're changing the line we might as well add the bracket.

Reply:
This is outdated now, so I assume this as "resolved"